packages/settingeditor/style/base.css

```diff
@@ -444,16 +461,32 @@ ul.jp-PluginList li.jp-mod-selected span.jp-PluginList-icon.jp-FileIcon {
   margin: 8px 12px 0;
 }
 
+.jp-PluginList mark {
+  background-color: transparent;
+  font-weight: bold;
+  color: var(--jp-ui-font-color1);
+}
+
 .jp-PluginList-entry {
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
   border: 1px solid transparent;
   background: transparent;
   overflow: hidden;
   padding: 4px 0 4px 4px;
   text-overflow: ellipsis;
   white-space: nowrap;
-  width: -webkit-fill-available;
+}
+
+.jp-PluginList-entry li {
+  margin-left: 27px;
+  margin-top: 5px;
+  color: var(--jp-ui-font-color1);
+  width: fit-content;
```

Comment:
```suggestion
  overflow-x: hidden;
  text-overflow: ellipsis;
```

| Before | After |
| --- | --- |
| ![Screenshot from 2022-03-20 00-47-07](https://user-images.githubusercontent.com/5832902/159143360-e82e497c-eb17-4432-b9ba-a2b0f1000875.png) | ![Screenshot from 2022-03-20 00-46-41](https://user-images.githubusercontent.com/5832902/159143361-bc476d50-652c-4ac6-add8-69d2f9b90a64.png) |